### PR TITLE
Adjust theme

### PR DESCRIPTION
**Pull request by dignite**
# What

- Adjusted colors
- Changed font
- Configure icon animation
- Pretty checkboxes
- Make sure config window layout does not collapse when window too narrow

# Running

_Master_
image

_This branch (old)_
image

_This branch (updated)_
image

# Flashing

_Master_
image
image

_This branch (old)_
image
image

_This branch (updated)_
image
image

# Paused

_Master_
image

_This branch (old)_
image

_This branch (updated)_
image

# Configuration window

_Master_
image

_This branch (old)_
image

_This branch (updated)_
image

# Fullscreen

_Master_
image

_This branch (old)_
image

_This branch (updated)_
image

### CODE REVIEW

File: src/windows/config/index.html [hunk 1]
```diff
@@ -1,79 +1,108 @@
...
-    <div class="container">
-      <div class="time">
-        <label for="minutes">
-          Each turn lasts:
-          <input type="number" id="minutes" min="1" class="minutes" /> minutes
+
+<head>
+  <meta charset="utf-8">
+  <title>Mob Timer Configuration</title>
+  <link rel="stylesheet" href="../../../node_modules/@ibm/plex/css/ibm-plex.min.css">
```
Comment (niklas-r) [inline src/windows/config/index.html]: Do we need to include the whole CSS from this package in order to get the fonts, or do we use more stuff from here?

Comment (niklas-r) [reply src/windows/config/index.html]: I guess this doesn't affect the app that negatively since it's already bundled though.

**Review by niklas-r — APPROVED** [review]
Great with new colors to differentiate from the original repo. I like the toggle elements more than the checkboxes.

The screenshots helped a lot in this PR :)

File: src/windows/config/index.html (hunk 1 above)
Comment (dignite) [reply src/windows/config/index.html]: Great input, I will switch to a font that isn't that big. The resulting mob timer zip got way bigger with IBM plex.

**Pull request merged**